RUFAS/task_manager.py

```diff
@@ -301,8 +301,15 @@ def _run_tasks(
             self.task, produce_graphics=produce_graphics, metadata_depth_limit=metadata_depth_limit
         )
         results = self.pool.imap(task_with_args, single_run_args)
+        failed = []
         for _ in results:
-            pass
+            if _ is not None:
+                failed.append(_)
+
+        if len(failed) > 0:
+            info_map = {"class": TaskManager.__name__, "function": TaskManager._run_tasks.__name__}
+            om = OutputManager()
+            om.add_log("Not all tasks are successful", f"There are tasks failed, the tasks are:{failed}", info_map)
```

Comment:
```suggestion
            om.add_log("Not all tasks are successful", f"There are tasks failed, the tasks are: {failed}", info_map)
```
I suggest "Task(s) failed", "Failed tasks are: {failed}" - I think this would be a bit more readable and a little less redundant. Also, I think it would be better to have this be an error rather than a log.